CSS/Bootstrap edits to the login and signup pages

owenbochner opened the pull request:
Added some beautification aesthetics with bootstrap/css. Changed structure of html documents. Still needs further editing to notify user if they've typed passwords that don't match on signup page.

owenbochner committed:
Created some css and html edits

Added some beautification aesthetics with bootstrap/css. Changed structure of html documents. Still needs further editing to notify user if they've typed passwords that don't match on signup page.

CaseyJ2000 commented:
Looks good but the files need to be in the templates folder.

CaseyJ2000 commented:
I just noticed that at the bottom of the screen it is showing the jinja for showing flashes.

owenbochner committed:
rearranged files

login, and signup css files are now located in static. login and signup html files are now located in templates.

owenbochner committed:
fixed login.html to post to correct location

owenbochner committed:
updated flash messages

CaseyJ2000 commented:
Everything we discussed in the discord has been fixed. I am merging it now.
